spec/factories/accounts.rb

```diff
@@ -0,0 +1,14 @@
+FactoryGirl.define do
+  trait :with_account_owner do
```

Comment:
Technically, the trait is global, so you could apply it to any other factory instantiation now. However, that would clearly break if you tried to `FactoryGirl.create(:user, :with_account_owner)`, but because it's global, that's why I called it `:with_account_owner` instead of the shorter `:with_owner`.